collectors/htlcs_collector.go

```diff
@@ -59,6 +64,24 @@ func newHtlcMonitor(router lndclient.RouterClient,
 			Name:      "failed_forwards",
 			Help:      "count fo failed forwards",
 		}),
+		resolutionTimeHistogram: prometheus.NewHistogram(
+			prometheus.HistogramOpts{
```

Comment:
Similar comment here re a vector to add in additional labels. One other useful labels for this one in particular (also applies to the others), would be adding the payment hash itself. This would let us track MPP usage, possibly probing usage, hash re-use, see how long MPP payments take to resolve on avg, etc, etc.

Reply:
Added some labels here. We don't have payment hash rn, because it was a bit involved to surface that info in the original lnd PR. Also wondering whether labels would be the best way to track something as variable as payment hash?

Reply:
So I'm thinking the payment hash would only be used to group items in aggregation operators. So you could do something like `count(resolution_ms) > 1 by (payment_hash)` (assuming it's a gauge) which would let you track MPP usage over time as they're identified by repeated payment hashes.

Reply:
It could end up being rather unscalable though since one may end up with so many labels over time.